### ISSUE TITLE
I want to do DetectAcitivty Demo what Can I do ?

### ISSUE THREAD

**Issue by helloworkcupid** [amitshekhariitbhu/Android-TensorFlow-Lite-Example#4]
if I have a XXXX.pb File what is DetecotClasiter ，plese  tell me  what can    i do ?

**Comment by anonym24** [comment]
yes object detection would be much interesting than classification
https://github.com/tensorflow/models/tree/master/research/object_detection

could some optimize it for Tensorflow Lite/NNAPI?
